api/client/commands.go

```diff
@@ -1823,6 +1823,33 @@ func (cli *DockerCli) CmdLogs(args ...string) error {
 	return cli.streamHelper("GET", "/containers/"+name+"/logs?"+v.Encode(), env.GetSubEnv("Config").GetBool("Tty"), nil, cli.out, cli.err, nil)
 }
 
+func (cli *DockerCli) CmdModify(args ...string) error {
+	cmd := cli.Subcmd("modify", "CONTAINER ACTION ARGUMENTS", "Modify the state of a running container")
```

Comment:
cmd := cli.Subcmd("modify", "CONTAINER ACTION ARGUMENTS", "Modify the state of a running container",true)